apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json

```diff
@@ -115,6 +123,120 @@
+                            "properties": {
+                                "id": {
+                                    "title": "id",
+                                    "type": "string"
+                                },
+                                "index": {
+                                    "title": "index",
+                                    "type": "number"
+                                },
+                                "name": {
+                                    "title": "name",
+                                    "type": "string"
+                                },
+                                "type": {
+                                    "enum": [
+                                        "floor"
+                                    ],
+                                    "title": "type",
+                                    "type": "string"
+                                },
+                                "units": {
+                                    "items": {
+                                        "additionalProperties": true,
+                                        "properties": {
+                                            "id": {
+                                                "title": "id",
+                                                "type": "string"
+                                            },
+                                            "label": {
+                                                "title": "label",
+                                                "type": "string"
+                                            },
+                                            "name": {
+                                                "title": "name",
+                                                "type": "string"
+                                            },
+                                            "type": {
+                                                "enum": [
+                                                    "unit"
+                                                ],
+                                                "title": "type",
+                                                "type": "string"
+                                            },
+                                            "unitType": {
+                                                "enum": [
+                                                    "flat",
+                                                    "parking"
+                                                ],
+                                                "title": "unitType",
+                                                "type": "string"
+                                            }
+                                        },
+                                        "required": [
+                                            "id",
+                                            "label",
+                                            "type"
```

Comment:
What about `unitType` as required property?

Reply:
@AntonAL this field can't be required by backward compabality issues. I decided not to run "fix" script on production database because first of all it is a bad practice and secondary anyway we only can add `flat` type to all current units. So I think better solution here is just leave this field not required